packages/patternfly-4/react-integration/demo-app-ts/src/components/demos/NotificationBadgeDemo/NotificationBadgeDemo.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { NotificationBadge, NotificationBadgeProps } from '@patternfly/react-core';
+import { BellIcon } from '@patternfly/react-icons';
+
+interface NotificationBadgeDemoState {
+    isRead: boolean;
+}
+
+export class NotificationBadgeDemo extends React.Component <NotificationBadgeProps, NotificationBadgeDemoState>{
+  constructor(props) {
+    super(props);
+    this.state = {
+      isRead: false
+    };
+  }
+  onClick = () => {
+    this.setState({
+      isRead: true
```

Comment:
Could go with `isRead: !this.state.isRead` to allow toggling.

Reply:
Thanks @redallen I figured once you set it you wouldn't want to show unread again, but if we want I can update the example to have it toggle. Just let me know I'm fine either way.